src/components/10-atoms/checkbox/index.js

```diff
@@ -218,7 +232,12 @@ class AXACheckbox extends NoShadowDOM {
           </label>
         `
       : html`
-          <label for="${refId}" class="a-checkbox__wrapper">
+          <label
+            for="${refId}"
+            class="a-checkbox__wrapper"
+            tabindex="0"
```

Comment:
Is this needed?

I thought using `tabindex` is dangerous since it can lead to unwanted inconvenience in combination with other components who can be addressed with the tabulator (You can skip whole elements, [example](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_global_tabindex) ).

I might be wrong but I would at least test it with other components

Reply:
Yeah it can be misused. In this case it is fine, because the label contains the input element and because I use a tabindex of 0 instead of any positive value (that is mostly being warned about).